### DISCUSSION TITLE
Research papers on Compilers

### DISCUSSION THREAD

**Discussion by LoopGlitch26** [hackclubiter/onEditor#10] [Resources]
Here we'll share some published articles and papers that we collected for research during our **research term** of this project

**Comment by Saswat7101** [comment]
For starters, I think [this](http://ijirt.org/master/publishedpaper/IJIRT100158_PAPER.pdf) looks promising. Just to have an overall idea. Here we can read what are the basic things we need to do to design a compiler from scratch.

**Comment by Saswat7101** [comment]
Compiler Basic Design and Construction
[Link](https://ijcsmc.com/docs/papers/October2014/V3I10201482.pdf)

**Reply by saswatsamal** [reply]
Please add any title for it so that we can understand before going into it.👍

**Reply by Saswat7101** [reply]
Okk will do

**Comment by saswatsamal** [comment]
Listing down some research papers for online compiler: 
1. [Review Paper on Online Java Compiler - IRJET](https://www.google.com/url?sa=t&source=web&rct=j&url=https://www.irjet.net/archives/V4/i3/IRJET-V4I3122.pdf&ved=2ahUKEwj_o_b53ejvAhXlq0sFHVtMD4QQFjADegQICRAC&usg=AOvVaw31r7chV0FRARvSJzsiFffF&cshid=1617681715663)
2. [Online Editor for Compiling and Executing Different Languages](https://www.google.com/url?sa=t&source=web&rct=j&url=http://ijarcsse.com/Before_August_2017/docs/papers/Volume_6/3_March2016/V6I3-0172.pdf&ved=2ahUKEwj_o_b53ejvAhXlq0sFHVtMD4QQFjAEegQIEhAC&usg=AOvVaw3_9cLiqRgp8qc16q1xc93O&cshid=1617681892578)

**Comment by LoopGlitch26** [comment]
[JudgeO](https://ieeexplore.ieee.org/document/9245310) research paper. It will be very handy to understand a high level approach to the system
